Model/Observer.php

```diff
@@ -14,57 +16,39 @@
  */
 class Observer implements ObserverInterface
```

Comment:
Just a side note - for our next minor release we should move this under the `Observer` namespace and give it a better name.

Reply:
understood, added this to my todo list ;)